examples/preflight/mysql.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: troubleshoot.replicated.com/v1beta1
+kind: Preflight
+metadata:
+  name: sample
+spec:
+  collectors:
+    - mysql:
+        collectorName: my-mysql
+        uri: postgresql://doadmin:nokq949a9dom9qyx@postgres-test-do-user-1634526-0.a.db.ondigitalocean.com:25060/defaultdb?sslmode=require
```

Comment:
this looks a little more 'real' than the others

Reply:
:+1: deleted and removed.  this was a e2e test (empty) server, shouldn't have been added but no worries if this connection string is out